src/components/CovidHome/Home.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+// import { useDispatch } from 'react-redux';
+import home from '../../images/petri-dish-removebg-preview.png';
+import styles from './Home.module.css';
+// import { fetchCovidCases } from '../../redux/covid/covid';
+
+function Home() {
+  return (
+    <div className={styles.homeContainer}>
+      <div className={styles.text}>
+        <h3> A Statistics of covid cases in the world</h3>
+      </div>
+      <div className={styles.picture}>
+        <img src={home} className={styles.img} alt="covid" />
+      </div>
+    </div>
+  );
+}
+
+export default Home;
```

Comment:
- [ ] Please the Home page has a list of categories that could be filtered using given parameters, e.g. category name.
A Search Input and clickable search button  to filter by name

- [ ] Please remove all commented code from your codebase to make it clean and easy to maintain

Reply:
Hello @CollinsTatang, 

Thank you for the review but the search input feature Is not part of the curriculum requirements. Please review the requirements again. 

Thank you.

Reply:
Hi @summyalena 
Thank you for your kind replied 
Please we have it under your homepage [description](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md)
![search](https://user-images.githubusercontent.com/55596970/194528663-a08b0aa7-b5ec-4cb8-8a0f-3745c6421138.png)